lib/active_record/connection_adapters/cockroachdb_adapter.rb

```diff
@@ -617,6 +532,26 @@ def load_additional_types(oids = nil)
           super
         end
 
+        def load_types_queries_with_aost(initializer, oids)
+          query = <<~SQL
+            SELECT t.oid, t.typname, t.typelem, t.typdelim, t.typinput, r.rngsubtype, t.typtype, t.typbasetype
+            FROM pg_type as t
+            LEFT JOIN pg_range as r ON oid = rngtypid AS OF SYSTEM TIME '-10s'
+          SQL
+          if oids
+            yield query + "WHERE t.oid IN (%s)" % oids.join(", ")
+          else
+            yield query + initializer.query_conditions_for_known_type_names
+            yield query + initializer.query_conditions_for_known_type_types
+            yield query + initializer.query_conditions_for_array_types
+          end
+        end
+
+        def configure_connection
+          super
+          internal_execute("SET intervalstyle_enabled = true", "SCHEMA")
```

Comment:
I'm not sure we need it, I blindly followed recommendation of a previous comment. This might not be necessary anymore (depending on how crdb's configuration is)

Reply:
we shouldn't need this now; it was only needed on v22.1 and earlier: https://github.com/cockroachdb/cockroach/pull/82388

Reply:
Weirdly the `NOTICE` about its usage still shows up in tests. Do you know what could be the cause ? Or how to track which query triggered a `NOTICE` event ?